packages/@aws-cdk/aws-ecr/lib/repository-ref.ts

```diff
@@ -28,21 +112,36 @@ export abstract class RepositoryRef extends cdk.Construct {
   public abstract addToResourcePolicy(statement: iam.PolicyStatement): void;
 
   /**
-   * Export this repository from the stack
+   * The URI of this repository (represents the latest image):
+   *
+   *    ACCOUNT.dkr.ecr.REGION.amazonaws.com/REPOSITORY
+   *
    */
-  public export(): RepositoryRefProps {
-    return {
-      repositoryArn: new cdk.Output(this, 'RepositoryArn', { value: this.repositoryArn }).makeImportValue().toString(),
-    };
+  public get repositoryUri() {
```

Comment:
Do we need both this method, and `repositoryUriForTag`?

Reply:
I think `repo.repositoryUri` is useful on it's own.

Reply:
Meh. I would just have `repositoryUri` take an optional `tag` argument and call it a day, but do as you wish.